content/issues/tracking-your-work-with-issues/about-tasklists.md

```diff
@@ -45,7 +45,7 @@ You can create a tasklist using Markdown in the issue description (the opening c
 1. To begin your tasklist, type <code>```[tasklist]</code> (triple backticks and <code>tasklist</code> inside square brackets) on a new line in the issue description.
 1. Optionally, type `### TITLE` on the next line, replacing `TITLE` with a title for your tasklist.
 1. For each item you want to add to your tasklist, type `- [ ]` on a new line, followed by a space, and either a link to an issue, a link to a pull request, or some text to create a draft issue.
-   - You must provide a full link to an issue or pull request. For example, `https://github.com/octo-org/octo-repo/issues/45`.
+   - You may provide a full link to an issue or pull request, or use the shorthand syntax. For example, `https://github.com/octo-org/octo-repo/issues/45` or `#45`.
```

Comment:
```suggestion
   - You may provide a full link to an issue or pull request, or use the shorthand `#ISSUE_NO` syntax. For example, `https://github.com/octo-org/octo-repo/issues/45` or `#45`.
```

Maybe this is clearer? 🤔 @stevecat defer to you here

Reply:
I really like including this! I made one further change to your suggestion, Omer, to make it clear that everything needs to be in the same repo in order to use the shorthand syntax.

```suggestion
   - You may provide a full link to an issue or pull request or, if the issue or pull request is in the same repository as your tasklist, you can use the #ISSUE-NUMBER shorthand syntax. For example, `https://github.com/octo-org/octo-repo/issues/45` or `#45`.
```

I'll apply this, approve the PR, and get this merged.